### ISSUE TITLE
Make NU1101 more elaborate to include commands for examining and fixing/adding the required sources

### ISSUE THREAD

**Issue by anangaur** [NuGet/docs.microsoft.com-nuget#2886]
As part of this issue: https://github.com/dotnet/maui/issues/7974 , the user looked at the NU1101 docs but could not figure what needed to be done. Hence this issue is to improve the docs to make it action driven than verbose comments with indirections to fix the issue.

In the solution, we may want to add
1. Command to examine the list of sources 
2. Command to add a specific source with the example of how to add nuget.org as a source, if not present. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c334f18d-66ca-28a9-3350-c20e762bbec1
* Version Independent ID: 2ea96dfa-e016-ae3b-864c-2a1ce513d90e
* Content: [NuGet Error NU1101](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1101)
* Content Source: [docs/reference/errors-and-warnings/NU1101.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/main/docs/reference/errors-and-warnings/NU1101.md)
* Product: **nuget**
* GitHub Login: @zhili1208
* Microsoft Alias: **lzhi**

**Comment by JonDouglas** [comment]
TL;DR:
 
https://github.com/chocolatey/choco/issues/2233
 

- We fixed a bug to do the right thing which others worked around for years.
- Powershell, Chocolatey, etc were using older NuGet versions.
- Customers may generate empty config if they use other programs prior to VS/dotnet CLI.
- We still are seeing the long tail of issues here and there.

We should add a note in NU1101 to tell the user where the user configuration is, how to delete it, and then use VS/dotnet CLI to generate a new one if they encounter this. And/Or empower them to add the central registry as a source.
